mock-kubernetes-client/client/client_test.go

```diff
@@ -99,12 +99,12 @@ var _ = Describe("[mock client]", func() {
 			Expect(len(pod.Spec.Containers)).To(Equal(1))
 			Expect(pod.Spec.Containers[0].Name).To(Equal("test-container"))
 			Expect(pod.ObjectMeta.Generation).To(Equal(int64(1)))
-			Expect(pod.Status.PodIP).To(Equal("1.1.1.1"))
+			Expect(pod.Status.PodIP).To(Equal("1.1.0.1"))
```

Comment:
The tests are failing because of this change.

Reply:
Changed the IP